e2e/transform/transform-runner/package.json

```
@@ -0,0 +1,11 @@
+{
+  "jest": {
+    "rootDir": "./",
+    "runner": "<rootDir>/runner.ts"
+  },
+  "dependencies": {
+    "@babel/preset-env": "^7.0.0",
+    "@babel/preset-typescript": "^7.0.0",
+    "jest-environment-node": "^26.6.2"
```

```suggestion
    "jest-environment-node": "file:../../../packages/jest-environment-node"
```

does that work? to avoid hard-coding.

---

do we even need these dependencies? I don't see `yarn` being run in here

---

I'll check, the changes were made before yarn berry came in, with the `file:` protocol I'll see if it works

---

It works with the `file:` protocol in my local, updating in both here and the `jest-env` PR